sonic-net/sonic-utilities
# Expanding the manifest capabilities

Expand docker manifest capabilities



#### What I did
Expanding the manifest.json file to include new network capabilities including the
ports forwarding and docker network type configuration: host, bridge, etc. 

#### How I did it
updating the manifest py file as well as the service_creatore py file.

#### How to verify it
In order to verify this feature, we need to have the complementary part in the sonic-buildimage repo
and then need to install a docker image file with manifest that includes ports in it and network type as well using sonic-package-manager and validate that they are as expected using docker commands such as: 'docker ps' and 'docker inspect' commands
